api/src/main/java/org/openmrs/api/VisitService.java

```diff
@@ -379,4 +379,57 @@
 	 */
 	@Authorized(PrivilegeConstants.EDIT_VISITS)
 	public void stopVisits(Date maximumStartDate);
+	
+	/**
+	 * @param visit is the vist to check 
+	 * @param location is the location of the vist
+	 * @param when is the time of the visit
+	 * @return true if when falls in the visits timespan AND location is within visit.location
+	 * @since 2.5.0 
+	 * <strong>Should</strong> return false if when is either less than VisitStartTime or greater than VisitEndTime 
```

Comment:
Remove all empty spaces that you are leaving between the comment sections

Reply:
hello @sherrif10 
 I have removed all the spaces between method comments.  I request for review

Reply:
Thanks for your response cc @dkayiwa